apps/s_server.c

```diff
@@ -131,7 +131,7 @@ static unsigned int psk_server_cb(SSL *ssl, const char *identity,
     if (s_debug)
         BIO_printf(bio_s_out, "psk_server_cb\n");
 
-    if (SSL_version(ssl) >= TLS1_3_VERSION) {
+    if (SSL_version(ssl) >= TLS1_3_VERSION && SSL_version(ssl) < DTLS1_2_VERSION) {
```

Comment:
Typically we would use `DTLS_VERSION_{GT,GE,LT,LE}` for version comparisons with DTLS versions.

Reply:
@kaduk Is it ok to add+use `ssl/ssl_local.h` in s_server?

Reply:
Best not to include that one.  It's not public and the apps are meant to only use public stuff.

Reply:
Ah, you're right; I forgot this was in `s_server`.